### PR TITLE
Publish docs for previous release versions

### PR DESCRIPTION
**Pull request by bestbeforetoday**
Contributes to #4300 

For builds of versions other than the latest release version, build
docs using the major and minor version as the target docs directory,
e.g. `v0.19`.

Signed-off-by: Mark S. Lewis <mark_lewis@uk.ibm.com>

### CODE REVIEW

File: .travis/script.sh [hunk 1]
```diff
@@ -39,46 +39,25 @@ if [ "${DOCS}" != "" ]; then
...
-            npm run linkcheck:next
-        elif [[ "${BUILD_FOCUS}" == "v0.16" ]]; then
-            npm run full:v0.16
-            npm run linkcheck:v0.16
-        elif [[ "${BUILD_FOCUS}" == 'v0.19' ]]; then
-            npm run full:v0.19
-            npm run linkcheck:v0.19
-        else 
-            _exit "Unknown build focus" 1 
+    if [ "${BUILD_RELEASE}" = 'unstable' ]; then
```
Comment (davidkel) [inline .travis/script.sh]: Wrong quotes used RHS

File: .travis/script.sh [hunk 2]
```diff
@@ -39,46 +39,25 @@ if [ "${DOCS}" != "" ]; then
...
+        if [ "${BUILD_FOCUS}" = 'latest' ]; then
+            BUILD_LABEL='unstable'
+        else
+            BUILD_LABEL="${BUILD_FOCUS}-unstable"
         fi
-
+    elif [ "${BUILD_RELEASE}" = 'stable' ]; then
```
Comment (davidkel) [inline .travis/script.sh]: Different quotes RHS

File: .travis/script.sh [hunk 3]
```diff
@@ -39,46 +39,25 @@ if [ "${DOCS}" != "" ]; then
...
+        BUILD_LABEL="${BUILD_FOCUS}"
     else
-       _exit "Unkown build release or focus ${BUILD_RELEASE} ${BUILD_FOCUS}" 1
+       _exit "Unkown build release: ${BUILD_RELEASE}" 1
```
Comment (davidkel) [inline .travis/script.sh]: I see a Matthew type spelling mistake here itching to be fixed

File: .travis/script.sh (hunk 1 above)
Comment (bestbeforetoday) [reply .travis/script.sh]: No variable expansion wanted on the RHS so single quotes seem appropriate.

File: .travis/script.sh [hunk 4]
```diff
@@ -39,46 +39,25 @@ if [ "${DOCS}" != "" ]; then
...
-        elif [[ "${BUILD_FOCUS}" == "v0.16" ]]; then
-            npm run full:v0.16
-            npm run linkcheck:v0.16
-        elif [[ "${BUILD_FOCUS}" == 'v0.19' ]]; then
-            npm run full:v0.19
-            npm run linkcheck:v0.19
-        else 
-            _exit "Unknown build focus" 1 
+    if [ "${BUILD_RELEASE}" = 'unstable' ]; then
+        if [ "${BUILD_FOCUS}" = 'latest' ]; then
```
[the remark this answers is not in the document]
Comment (bestbeforetoday) [reply .travis/script.sh]: Not sure why this would fail?

File: .travis/script.sh (hunk 3 above)
Comment (bestbeforetoday) [reply .travis/script.sh]: Agreed! :)

**Pull request merged**